### ISSUE TITLE
Component Controls missing

### ISSUE THREAD

**Issue by Mrjaco12** [spartansystems/atomic_cms#11]
If a component is missing items for cms fields, then none of the controls are available to modify its position or its presence on the page.

**Comment by bbrock25** [comment]
So, this is just for blank components that can be included anywhere?

**Comment by Mrjaco12** [comment]
Yeah, it's for components that don't require any options. But you may still want the controls in the CMS editor so you can adjust position or remove.
